cognite/client/_api/functions.py

```diff
@@ -729,12 +729,12 @@ def _validate_and_parse_requirements(requirements: List[str]) -> List[str]:
     return parsed_reqs
 
 
-def _write_requirements_to_file(file_path: str, requirements: List[str]) -> None:
-    with open(file_path, "w+") as f:
-        f.write("\n".join(requirements))
+def _write_requirements_to_file(file: IO, requirements: List[str]) -> None:
+    if not file.closed:
```

Comment:
What about the scenario where the file is closed? Is that not possible?

Reply:
Okay. I'll remove the if :)